js_minimizer is not callable

The following run function of JSMinimizer class is not callable as `js_minimizer` is an object and not a function, thus it should not be callable.

```
    js_tokenizer = AntlrTokenizer(JavaScriptLexer)

    js_minimizer = JSMinimizer(
        utils.test,
        max_threads=thread_count,
        tokenizer=js_tokenizer.tokenize,
        token_combiner=js_tokenizer.combine,
        file_extension=file_extension)

    result = line_minimizer.minimize(data)
    result = js_minimizer(result)
    result = js_minimizer(result)
    result = line_minimizer.minimize(result)
```

Location in codebase [here](https://github.com/google/clusterfuzz/blob/master/src/python/bot/minimizer/js_minimizer.py#L187).

Also, does js_minimizer should be called two times?
I guess `js_minimizer(result)` should be replaced with `js_minimizer.minimize(data)`, but I am not really sure. Can someone please confirm if it is a bug or not?

I got the error while using pylint to check this file for any linting issues:

```
************* Module src.python.bot.minimizer.js_minimizer
bot/minimizer/js_minimizer.py:187:13: E1102: js_minimizer is not callable (not-callable)
bot/minimizer/js_minimizer.py:188:13: E1102: js_minimizer is not callable (not-callable)
```

--- a/src/python/bot/minimizer/js_minimizer.py
+++ b/src/python/bot/minimizer/js_minimizer.py
@@ -184,8 +184,8 @@ class JSMinimizer(minimizer.Minimizer):
         file_extension=file_extension)
 
     result = line_minimizer.minimize(data)
-    result = js_minimizer(result)
-    result = js_minimizer(result)
+    result = js_minimizer.minimize(result)
+    result = js_minimizer.minimize(result)
     result = line_minimizer.minimize(result)
 
     return result

